src/rules/require-hook.ts

```diff
@@ -27,12 +31,28 @@ const isNullOrUndefined = (node: TSESTree.Expression): boolean => {
   );
 };
 
-const shouldBeInHook = (node: TSESTree.Node): boolean => {
+const isExcludedFnCall = (
+  node: TSESTree.CallExpression,
+  options: RequireHooksOptions,
+): boolean => {
+  const nodeName = getNodeName(node);
+
+  if (nodeName === null) {
+    return false;
+  }
+
+  return !!options.excludedFunctions?.includes(nodeName);
+};
+
+const shouldBeInHook = (
+  node: TSESTree.Node,
+  options: RequireHooksOptions,
```

Comment:
Since we currently don't have any other options, I think we might as well just pass in an string array as that's more expressive and it's not a lot of work to refactor if we add more options in the future

(it also means you don't have to build an object when calling this method)

Reply:
@G-Rath thanks. Also I got rid of `RequireHooksOptions` interface.